### PR TITLE
Translate news from April 13 (bg)

### PR DESCRIPTION
**Pull request by gmitrev**

### CODE REVIEW

File: bg/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: bg
+---
+
+Радваме се да обявим излизането на Ruby 2.0.0-p645.
+
+Включена е кръпка за сигурността за OpenSSL. Моля погледнете следната
+дискусия за повече детайли:
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline bg/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: Update title and link to `bg` post?

File: bg/news/_posts/2015-04-13-ruby-2-1-6-released.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: bg
+---
+
+Излезе Ruby 2.1.6.
+
+Включена е кръпка за сигурността за OpenSSL. Моля погледнете следната
+дискусия за повече детайли:
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline bg/news/_posts/2015-04-13-ruby-2-1-6-released.md]: Update title and link to `bg` post?

File: bg/news/_posts/2015-04-13-ruby-2-2-2-released.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+lang: bg
+---
+
+Радваме се да обявим излизането на Ruby 2.2.2.
+Това е бъгфикс рилийз към стабилната 2.2 серия
+
+Включена е кръпка за сигурността за OpenSSL. Моля погледнете
+темата по-долу за повече детайли.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline bg/news/_posts/2015-04-13-ruby-2-2-2-released.md]: Update title and link to `bg` post?

File: bg/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+layout: news_post
+title: "CVE-2015-1855: Ruby OpenSSL Hostname Verification"
```
Comment (JuanitoFatas) [inline bg/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: Translate title?

**Comment by gmitrev** [comment]
@JuanitoFatas Thanks for the feedback :)

**Pull request merged**